targets/psa_switch/externs/psa_checksum.cpp

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
missing apache copyright header

Reply:
Yes, I am sorry. I will add it.

Reply:
are you making this contribution on behalf of Cornell University? I'm asking because that's the copyright you are using but it doesn't match your email address / Github profile.

On a side note, I also found out recently that `<YEAR>-present` is not a valid mention, so please just use `YEAR`

Reply:
Thank you for paying me attention to this. I am making this contribution on behalf of SYRMIA LCC. I have just changed the copyright.